res/lang/gen_settings/genSettings_en.txt

```diff
@@ -50,17 +50,17 @@ MUSIC                       = Music
 SFX                         = Sound effects
 FULLSCREEN                  = Fullscreen
 VSYNC                       = VSync
-RIGHTCLICK_EMULATION              = Emulate "right click"
-RIGHTCLICK_EMULATION_EXPLANATION  = Emulate right-click with long-down-click of left mouse button or touch
-RIGHTCLICK_EMULATION_TIME         = Time before "right click"
-WINDOW_MODE_RESOLUTION      = Window mode resolution
+RIGHTCLICK_EMULATION              = Emulate right-click
+RIGHTCLICK_EMULATION_EXPLANATION  = Emulate right-click with long-press of left mouse button or touch
+RIGHTCLICK_EMULATION_TIME         = Time before right-click
+WINDOW_MODE_RESOLUTION      = Windowed mode resolution
```

Comment:
In IT-Terms this is a "window resolution" not a "windowed resolution".

Reply:
I don't agree, the game is talking about "window mode", not "window resolution".  "Window" applies to the noun "mode", and in that context it's "fullscreen mode" vs "windowed mode".  See for example https://www.dosbox.com/ referring to "locking the mouse in windowed mode", or this reference to windowed vs fullscreen: https://tasvideos.org/Bizhawk/DisplayConfig#Window

Reply:
Accepted.